lib/govuk_publishing_components/app_helpers/asset_helper.rb

```diff
@@ -87,7 +87,7 @@ def is_already_used?(component)
       end
 
       def viewing_component_guide?
-        request.path.include?("/component-guide")
+        request&.path&.include?("/component-guide")
```

Comment:
I'm not familiar with this syntax but I assume it does what it's supposed to! Is there any way there could be a test written for this?

Reply:
This was new to me in Ruby/Rails as well, there is further documentation on this here - https://ruby-doc.org/core-2.6/doc/syntax/calling_methods_rdoc.html#:~:text=Safe%20navigation%20operator%C2%B6%20%E2%86%91,if%20the%20call%20is%20skipped.

I should be able to add a test for this, will add this now.

Reply:
I've added a test to check what happens when the request object is not available - https://github.com/alphagov/govuk_publishing_components/pull/3455/commits/7725991e23b74ddcc9b298475752ad5706111b31